redbot/core/core_commands.py

```diff
@@ -457,11 +466,68 @@ def get_bot_uptime(self, *, brief: bool = False):
             await ctx.send("No exception has occurred yet")
 
     @commands.command()
-    @checks.is_owner()
-    async def invite(self, ctx: commands.Context):
+    @commands.check(CoreLogic._can_get_invite_url)
+    async def invite(self, ctx):
         """Show's Red's invite url"""
         await ctx.author.send(await self._invite_url())
 
+    @commands.group()
+    @checks.is_owner()
+    async def inviteset(self, ctx):
+        """Setup the bot's invite"""
+        pass
+
+    @inviteset.command()
+    async def public(self, ctx, confirm: bool = False):
+        """
+        Define if the command should be accessible\
+        for the average users.
+        """
+        if not self.bot.user.bot:
```

Comment:
Red doesn't have selfbot support, this condition should be removed.

Reply:
Should the condition on line 268 also be removed? It was already there when I made the branch I think.

Reply:
You should probably update your PR branch so you can more accurately view differences. That section isn't there on current code, and was removed prior to 3.0.0